android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -675,6 +696,14 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
         terminal.confirmRefund(RNRefundCallback(promise))
     }
 
+    @OptIn(OfflineMode::class)
+    fun getOfflineStatus(): ReadableMap = nativeMapOf {
+        val mutableMap = terminal.offlineStatus.sdk.offlinePaymentAmountsByCurrency.toMutableMap()
+        val sdkMap = mutableMapOf("offlinePaymentsCount" to terminal.offlineStatus.sdk.offlinePaymentsCount,
+            "offlinePaymentAmountsByCurrency" to mutableMap)
+        putString("sdk", sdkMap.toString())
```

Comment:
Fwiw, I think we might as well add the "reader" to this map as well. The iOS and Android SDKs will both expose both "sdk" and "reader" offline statuses, so we might as well translate both (even if we don't yet support sPOS readers offline yet).